contrib/inventory/cloudforms.py

```diff
@@ -174,6 +174,11 @@ def read_settings(self):
         else:
             self.cloudforms_nest_tags = False
 
+        if config.has_option('cloudforms', 'suffix'):
+            self.cloudforms_suffix = config.get('cloudforms', 'suffix')
```

Comment:
I'd like to see some sanity checking in here to ensure the resulting `self.cloudforms_suffix` value contains a `.` character at `self.cloudforms_suffix[0]` and raise an `AnsibleError` exception if it does not. Otherwise I think this looks good.

Reply:
Thanks @maxamillion - happy to accommodate that although my Python isn't the best :smile: Is it just a case of pretty much a `from ansible.errors import AnsibleError`, do the check and then `raise AnsibleError("Leading fullstop is required for Cloudforms Suffix")` if applicable?

Reply:
@rbywater that would be perfect, thanks!

Reply:
@maxamillion have actioned your review comments so ready for another review when you are ready :smile: